spec/requests/other_incomes_controller_spec.rb

```diff
@@ -102,7 +102,7 @@
 
         it "contains success false in the response body" do
           post_payload
-          expect(parsed_response).to match(errors: [/The property '#\/other_incomes\/1\/payments\/0' did not contain a required property of 'client_id' in schema file/], success: false)
+          expect(parsed_response).to match(errors: [/The property '#\/other_incomes\/1\/payments\/0' did not contain a required property of 'client_id' in schema/], success: false)
```

Comment:
@starswan It would be nice to use `expect(parsed_response[:errors])` with `include` or `match` for consistency